### ISSUE TITLE
Mismatched usage of self.vae_scaling_factor_image in i2v pipeline between pretrained model and model after lora fintuned.

### ISSUE THREAD

**Issue by OwalnutO** [a-r-r-o-w/finetrainers#155]
### System Info / 系統信息

Diffusers 0.32.1

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

In the official inference pipeline [pipeline_cogvideox_image2video.py](https://github.com/huggingface/diffusers/blob/v0.32.1/src/diffusers/pipelines/cogvideo/pipeline_cogvideox_image2video.py), line 385 to line 690,
```ruby     
if not self.vae.config.invert_scale_latents:
    image_latents = self.vae_scaling_factor_image * image_latents
else:
    # This is awkward but required because the CogVideoX team forgot to multiply the
    # scaling factor during training :)
    image_latents = 1 / self.vae_scaling_factor_image * image_latents
```
It suggests the the pretrained cogvideo model doesn't multiply the scaling factor, and the self.vae.config.invert_scale_latents is True in the provided [cogvideoX1.5-5B-I2V model ](https://huggingface.co/THUDM/CogVideoX1.5-5B-I2V/blob/main/vae/config.json).

However, in the lora finetuneing script [cogvideox_image_to_video_lora.py](https://github.com/a-r-r-o-w/finetrainers/blob/main/training/cogvideox/cogvideox_image_to_video_lora.py), line 743 to line 745
```ruby     
image_latents = image_latent_dist.sample() * VAE_SCALING_FACTOR
image_latents = image_latents.permute(0, 2, 1, 3, 4)  # [B, F, C, H, W]
image_latents = image_latents.to(memory_format=torch.contiguous_format, dtype=weight_dtype)
```
The vae_scaling_factor is correctly added. It seems that if we use lora finetuning script to finetune the pretrained I2V model, the usage of the self.vae_scaling_factor_image in the official inferece pipeline will be mismatched. I'm not sure wether this is an important problem and should be modified.

### Expected behavior / 期待表现

It seems that the official inferece pipeline should be modifed to be compatible with personal finetuned model.

**Comment by sayakpaul** [comment]
Yes, we need to account for this in the training script. I don't think the official pipeline implementation needs to be modified for this.

Note https://github.com/a-r-r-o-w/finetrainers/issues/136 as well. We will consider this when we start working on 136.  

Cc: @a-r-r-o-w
